lib/tasks/attempts.rake

```diff
@@ -77,9 +77,4 @@ namespace :attempts do
 
     puts 'Remember to restart Rails after updating application.yml.default!' if failed
   end
-
-  desc 'Clear all events from Redis'
-  task purge_events: :environment do
-    IrsAttemptsApi::RedisClient.clear_attempts!
-  end
```

Comment:
Oh shoot, @ThatSpaceGuy, we actually want this part, just not the other stuff I had added.

Reply:
Oops - sorry! I put it back now 😅